armi/reactor/components/component.py

```diff
@@ -636,6 +654,10 @@ def setNumberDensity(self, nucName, val):
         """
         Set heterogeneous number density.
 
+        .. impl:: Set number density of nuclide
+            :id: I_ARMI_COMP_NUCLIDE_FRACS0
+            :implements R_ARMI_COMP_NUCLIDE_FRACS
+
```

Comment:
Delete this one.

You can also ADD this one, if you like to `def setMassFracs()` in `composites.py`.  But definitely remove it here.